media/css/firefox/features/adblocker.scss

```diff
@@ -66,28 +57,15 @@ blockquote {
     }
 }
 
-.mzp-c-card-feature-media-wrapper {
-    margin-bottom: 50px;
-
-    @media #{$mq-lg} {
-        width: 100%;
-    }
-}
-
-.mzp-c-card-feature {
-    margin-bottom: 0;
-}
-
 section.mzp-c-hero {
     padding: 0;
+    .mzp-c-hero-subtitle {
+        @include text-display-sm;
+    }
 }
 
-.mzp-c-billboard-image {
-    width: 100%;
-}
-
-.section-title {
-    @include text-display-lg;
+h2.section-title {
```

Comment:
Does this need the `h2` qualifier? Is it just for higher specificity or are there other elements with the same class and we need to distinguish them?

Reply:
I had needed the higher specificity when playing with the headline styles, it's no longer necessary. :+1: